notebooks/samples/scala/Classification - AdultCensus.json

```diff
@@ -0,0 +1,683 @@
+      "dateCreated": "2019-05-24 16:27:22.156",
+      "dateStarted": "2019-05-24 16:29:41.641",
+      "dateFinished": "2019-05-24 16:29:41.647",
+      "status": "FINISHED",
+      "progressUpdateIntervalMs": 500
+    },
+    {
+      "text": "%spark.dep\nz.load(\"Azure:mmlspark:0.17\")",
+      "user": "admin",
+      "dateUpdated": "2019-05-24 15:33:54.717",
+      "config": {
+        "colWidth": 12.0,
+        "fontSize": 9.0,
+        "enabled": true,
+        "results": {},
+        "editorSetting": {
+          "language": "scala",
+          "editOnDblClick": false,
+          "completionKey": "TAB",
+          "completionSupport": true
+        },
+        "editorMode": "ace/mode/scala"
+      },
+      "settings": {
+        "params": {},
+        "forms": {}
+      },
+      "results": {
+        "code": "SUCCESS",
+        "msg": [
+          {
+            "type": "TEXT",
+            "data": "res0: org.apache.zeppelin.dep.Dependency \u003d org.apache.zeppelin.dep.Dependency@ad2ed0f\n"
+          }
+        ]
+      },
+      "apps": [],
+      "jobName": "paragraph_1558657539798_1492374222",
+      "id": "20190523-172539_613186693",
+      "dateCreated": "2019-05-23 17:25:39.798",
+      "dateStarted": "2019-05-24 15:33:54.730",
+      "dateFinished": "2019-05-24 15:34:23.305",
+      "status": "FINISHED",
+      "progressUpdateIntervalMs": 500
+    },
+    {
+      "text": "%md\nThen we load in the packages we need.",
+      "user": "admin",
+      "dateUpdated": "2019-05-24 16:29:57.203",
+      "config": {
+        "colWidth": 12.0,
+        "fontSize": 9.0,
+        "enabled": true,
+        "results": {},
+        "editorSetting": {
+          "language": "markdown",
+          "editOnDblClick": true,
+          "completionKey": "TAB",
+          "completionSupport": false
+        },
+        "editorMode": "ace/mode/markdown",
+        "editorHide": true,
+        "tableHide": false
+      },
+      "settings": {
+        "params": {},
+        "forms": {}
+      },
+      "results": {
+        "code": "SUCCESS",
+        "msg": [
+          {
+            "type": "HTML",
+            "data": "\u003cdiv class\u003d\"markdown-body\"\u003e\n\u003cp\u003eThen we load in the packages we need.\u003c/p\u003e\n\u003c/div\u003e"
+          }
+        ]
+      },
+      "apps": [],
+      "jobName": "paragraph_1558740586130_100953689",
+      "id": "20190524-162946_1779279677",
+      "dateCreated": "2019-05-24 16:29:46.130",
+      "dateStarted": "2019-05-24 16:29:57.203",
+      "dateFinished": "2019-05-24 16:29:57.207",
+      "status": "FINISHED",
+      "progressUpdateIntervalMs": 500
+    },
+    {
+      "text": "import com.microsoft.ml.spark.{TrainClassifier,ComputeModelStatistics, TrainedClassifierModel}\nimport scala.io.Source\nimport org.apache.spark.sql.{Dataset, SparkSession}\nimport spark.implicits._\nimport org.apache.spark.sql.types._\nimport org.apache.spark.ml.classification.LogisticRegression",
```

Comment:
Seems like the import locations have changed. ie. com.microsoft.ml.spark.ComputeModelStatistics is now com.microsoft.ml.spark.train.ComputeModelStatistics and so on..